README.md

```diff
@@ -22,29 +22,49 @@
 skope-rules
 ===========
 
-Skope-rules is a Python machine learning module built on top of
+*Skope-rules* is a Python machine learning module built on top of
 scikit-learn and distributed under the 3-Clause BSD license.
 
-Skope-rules aims at learning logical, interpretable rules for "scoping" a target
+*Skope-rules* aims at learning logical, interpretable rules for "scoping" a target
 class, i.e. detecting with high precision instances of this class.
 
-Skope-rules is a trade off between the interpretability of a Decision Tree
+*Skope-rules* is a trade off between the interpretability of a Decision Tree
 and the modelization power of a Random Forest.
 
 
 .. image:: ressources/schema.png
 
 |
 
-One of the original contribution of the package is the use of a semantic deduplication tree.
-It ensures that rules are diversified by design :
+The previous figure explains that skope-rules trains a tree based bagging estimator 
```

Comment:
As illustrated in the above figure, the idea behind skope-rules is to fit a tree based ...